### DISCUSSION TITLE
Access to trigger Airplay and Chromecast functionality in the API

### DISCUSSION THREAD

**Discussion by PaTiToMaSteR** [vimeo/player.js#667] [Ideas]
### Expected Behavior
Have in the API access to trigger airplay and chromecast

### Actual Behavior
For customization purposes of my app, I re-design the controls and sync all events, however I cannot access the video element so I cannot trigger the native airplay or chromecast scripts. Will be great if you add those 2 simple events that you already have to trigger externally

### Steps to Reproduce
N/A
<!--

[Test](estadodealarma.app/@Destacados?v=474785133/307f3c44db)

**Comment by fisherinnovation** [comment]
Thanks for your feature request. We will review.

**Comment by ehababdo** [comment]
Waiting for this 😄

**Comment by andrehhtm** [comment]
Upvote

**Comment by arnegolemar** [comment]
Upvote

**Comment by bdougherty** [comment]
This was added about a year ago in #916, although for now it only works with AirPlay.
